website/static/js/pages/project-dashboard-page.js

```diff
@@ -207,7 +207,10 @@ $(document).ready(function () {
                 var truncatedText = $.truncate(renderedText, {length: 400});
                 markdownElement.html(truncatedText);
                 mathrender.mathjaxify(markdownElement);
+                markdownElement.css('display', 'inherit');
```

Comment:
Minor: you should be able to do `markdownElement.show()`, which is more readable and possibly more correct.